src/celery_prometheus/prometheus_bootstep.py

```diff
@@ -26,7 +27,11 @@ def add_prometheus_option(app: Celery) -> None:
     if celery_version.major < 5:
 
         def add_preload_arguments(parser: ArgumentParser) -> None:
-            parser.add_argument("--prometheus-collector-addr", default=None, help=help)
+            parser.add_argument(
+                "--prometheus-collector-addr",
+                default=os.environ.get("CELERY_PROMETHEUS_COLLECTOR_ADDR"),
```

Comment:
Default is still effectively None. This just allows for one more check for the address string as an environment variable before giving up. Going with a distinct name prefix to avoid potential naming collisions.

Reply:
```suggestion
                default=os.getenv("CELERY_PROMETHEUS_COLLECTOR_ADDR"),
```

Reply:
Sure. I'll amend.